README.md

```diff
@@ -48,8 +48,8 @@ this is a big hammer, you could instead just tell the CVO your operator should b
 Build and push your newly built image to a repo:
 ```
 $ make images
-$ docker tag openshift/origin-cluster-svcat-apiserver-operator:latest jboyd01/origin-cluster-svcat-apiserver-operator:xx
-$ docker push jboyd01/origin-cluster-svcat-apiserver-operator:xx
+$ docker tag registry.svc.ci.openshift.org/ocp/4.2:cluster-svcat-apiserver-operator docker.io/jmrodri/ocp-cluster-svcat-apiserver-operator:latest
```

Comment:
Nit: maybe just use `<account>` instead of `jmrodri` so that its more obvious that whoever is hacking should use their own account since they won't be able to push to yours?

Reply:
That's fine, I did it because it was easier to copy :)